.github/workflows/ci.yml

```
@@ -12,15 +12,17 @@ jobs:
     name: CI
 
     steps:
-    - name: Setup Python
-      uses: actions/setup-python@v4
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
       with:
-        python-version: 3.8
+        python-version: 3.10
```

Now I understand the quotes
```suggestion
        python-version: '3.10'
```